ubuntu_15.04.sh

```diff
@@ -50,6 +53,9 @@ cd PyTAPS-1.4/
 python setup.py --iMesh-path=$HOME/opt/moab --without-iRel --without-iGeom install --user
 cd ..
 # Install PyNE
+if [ -d pyne ] ; then
+    rm -rf pyne
+fi 
```

Comment:
@Shimwell - did you have plans to replace the `rm -f` with a warning/error msg and halt?  I'd prefer that solution, so I can make a separate PR if you can't get to it.